srcpkgs/qt5pas/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'qt5pas'
+pkgname=qt5pas
+version=2.6lazarus1.8.4
+revision=1
+wrksrc=lazarus
+build_style=gnu-makefile
+hostmakedepends="qt5-qmake"
+makedepends="qt5-x11extras-devel"
+short_desc="Pascal bindings for qt5"
+maintainer="mobinmob <mobinmob@disroot.org>"
+license="modified LGPL-2"
+homepage="https://www.lazarus-ide.org/"
+distfiles="https://sourceforge.net/projects/lazarus/files/Lazarus%20Zip%20_%20GZip/Lazarus%201.8.4/lazarus-1.8.4.tar.gz"
+checksum=5a9801ac341a322ad1fa104d373f154d8ab72ba0add1693d0dcc8d936e6c10e9
+nocross=yes
+
+do_build() {
+	cd "$wrksrc"/lcl/interfaces/qt5/cbindings/
```

Comment:
you can use the `qmake` build_style and set build_wrksrc to lcl/interfaces/qt5/cbindings

Reply:
the qmake  build_style will handle those things